hail/src/main/scala/is/hail/backend/local/LocalBackend.scala

```diff
@@ -341,6 +342,29 @@ class LocalBackend(val tmpdir: String) extends Backend with BackendWithCodeCache
       }
     }
 
+  def pyRegisterIR(
```

Comment:
This looks the same as the definition in SparkBackend. Any reason not to just define it once in the Backend base class?

Reply:
good call out, I'll do that